mantid-developer-linux.yml

```diff
@@ -0,0 +1,71 @@
+name: mantid
+
+channels:
+  - conda-forge
+  - DLR-SC
```

Comment:
Can I just check what we need this channel for? Can we add a comment about which package it is?

Reply:
OpenCascade, and will do!